libs/fn/src/lib/cdk/selectable-list/selectable-item.directive.ts

```diff
@@ -0,0 +1,117 @@
+import { Directive, ElementRef, EventEmitter, HostListener, Inject, Input, Optional, Output } from '@angular/core';
+import { merge, Observable } from 'rxjs';
+import { SelectableItemToken } from './SelectableItemToken';
+import { FN_SELECTABLE_ITEM_PROVIDER } from './selectable-list.tokens';
+import { FN_DISABLED } from '../tokens/disabled';
+import { DisabledBehavior } from '../interfaces/DisabledBehavior';
+import { ReadonlyBehavior } from '../interfaces/ReadonlyBehavior';
+import { FN_READONLY } from '../tokens/readonly';
+import { SelectionService } from './selection.service';
+
+@Directive({
+    selector: '[fnSelectableItem]',
```

Comment:
`fnSelectableItem` but `fnFocusable`, let's make them consistent.

Reply:
Which one would be better? Leave "item" postfix or nah?

Reply:
I was thinking about having also only [fnSelectable] in case of grouping you know. Every item could have been as well as root, as well as item. So that they would resolve children and add selection logic to them. On self selection it would select probably itself and all of it's children or something like that. Like a Tree

Reply:
I'd prefer with `item` postfix. This will allow us to add `fnSelectable`  for a group

Reply:
I was having some fun with tree data structure visualisation couple months ago. Check out my profile, it should be there. Same principles can be applied to these too

Reply:
My thought is that if we have `fnSelectableList` then we should have `fnSelectableItem` and the same for any similar cases. 

If we go with `list` and ` ` (nothing, i mean `fdSelectable`, without the suffix) there will be less coherence between them.

In the same time i don't see any issues if we will be using `fnSelectableItem` without the list, `item` name looks okay even without the container.

Reply:
Makes sense. In any ways we always can update. I will take these inco account in next push sequence

Reply:
`fnSelectable` sounds more common, when we will be discussing them we will say `selectable` & `selectable list`, there is a big chance to make a mistake, but when it's the `item` and `list` there are no changes to missunderstant it.